ddtrace/profiling/collector/memalloc.py

```diff
@@ -166,7 +166,7 @@ def collect(self):
         thread_id_ignore_set = self._get_thread_id_ignore_set()
 
         if self._export_libdd_enabled:
-            for (frames, nframes, thread_id), size, domain in events:
+            for (frames, nframes, thread_id), size, _domain in events:
```

Comment:
What is the problem here? Does `domain` take different types? This might be the sign of a potential bug.

Reply:
Ah it looks like we are not using the `domain` here. @sanchda does it resonate with you that we are currently not making use of the memory domain? Just to confirm that this is not a bug.